.github/workflows/GoogleJavaFormat.yml

```diff
@@ -16,5 +16,5 @@ jobs:
         uses: axel-op/googlejavaformat-action@v3
         with:
           skipCommit: true
-          version: 1.10.0
+          version: 1.15.0
```

Comment:
We deliberately added a version here 1.10.0 
Can you change wiki: https://snowflakecomputing.atlassian.net/wiki/spaces/EN/pages/1673103743/Kafka+Connector+Setup+Debugging if we need 1.15.0? 

Or you could revert it to 1.10.0 so that zekai and I dont have to update. Either Version is fine

Reply:
This is failing the gfmt too, might want to do it in separate PR though if you decide 1.15.0

Reply:
revert back to 1.10.0